### ISSUE TITLE
beta: beta1: dcblox: "LDR R1" Unit 2 RC10: (ID: BUG-0008) "CORTX Manager: LDR Web server failing to save or provide information"

### ISSUE THREAD

**Issue by cortx-admin** [Seagate/cortx-prvsnr#4778]
*CONTACTS*
 SE=Ricardo Alvarez
 Customer=mark.sprouse@dcblox.com

BACKGROUND
 * Correct Onboarding URL is
 "https://<MANAGEMENT_VIP>:28100/#/preboarding/welcome"

*PROBLEM STATEMENT*
 * Customer has completed Unboxing sequence under Eng/SE/Support supervision on MON:17/AUG/2020.
 * Customer may not have completed onboarding sequence using the correct Onboarding URL. SE to confirm.

*PROBLEM SYMPTOM*
 * RC10, CORTX Manager reporting error dialog box : -
 ** *"No configurations found. Please complete the onboarding process"*  <-: symptom of *this* ticket: ie. EOS-12283
 * RC10, CORTX Manager at end of screen  : -
 ** *"Root user already exists"* <-: symptom of *this* ticket: ie. EOS-12283
 * RC10, CORTX Manager reporting error dialog box : -
 ** *"Server is taking too long to respond. Please refresh the page"* <-: symptom of ticket: EOS-12329 ie EOS-12255.

*LIKELY RECREATION SEQUENCE*
 The Customer performed the following sequence which has likely lead to this situation :-
 1. User went to correct Unboarding URL.
 2. User entered "cortxub" as the CORTX Manager administrator account user name (which is illegal).(see corrective suggestions EOS-12225 and EOS-12241) User clicked on accept.
 3. Seagate asked user to go *BACKWARDS* and change the CORTX Manager administrator account to something else which the user complies. User then finished rest of Onboarding sequence.
 4. Whilst inside CORTX Manager main screens the red dialog "No configuration found. Please complete the onboarding process appears".

*CUSTOMER PROBLEM STATEMENT*
{code:java}
Steps to Reproduce:
1. Perform Unboxing script
2. Access the Web interface
3. Try to access some menus. 
4. Request will fail

Detailed Description:
See attached video to se behavior. 
{code}
*HISTORY*
 2020-08-19 STATUS: Customer may not have completed onboarding sequence using the correct Onboarding URL. red dialog box saying "No configuration found. Please complete onboarding process.". NEXT_STEP: Eng to review.
 2020-08-19 STATUS: Customer now provided with correct onboarding URL but has hit a red dialog box saying "Root user already exists". Ivan has suggested a sequence to get feedback from Eng. NEXT_STEP: Eng to feedback on steps to recover and hence allow user to re-run onboarding (if needed).
2020-08-20 STATUS: Eng recommend Eng attend a customer call asap at customer convenience to get them going. Eng confirmed that "cortxub" is a valid CSM administrator account name. CSM logs needed, but will be examined by Eng on the customer facing call. NEXT_STEP: Asking SE to arrange a customer call with [~720600] and Support.

2020-08-20 STATUS: Eng, SE and Support attended customer call where this ticket EOS-12283 (not completing unboxing due to customer confusion) and separate issue EOS-12329 (kibana and elasticSearch on serverB is not functional) were both resolved on site. Method used for former is to talk customer through the proper URL. Method used for later is to verify the problem and then restart the server B kibana and elasticSearch. It remains unexplained why pacemaker says everything is fine after unboxing but infact server B had service issues. Customer now unblocked and able to perform S3 traffic. NEXT_STEP: For this ticket EOS-12283, Eng to evaluate if a CORTX Manager change is needed to prevent the user navigating away from Preboarding URL (aka Onboarding in Documentation) until it is complete. For EOS-12329, Eng to evaluate mechanisms by which kibana and elasticSearch can become faulted and also why when faulted, they are not detected as faulted by pacemaker and auto corrected.
